src/components/TabComponent/TabComponent.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from 'react';
+import './TabComponent.css';
+
+export default class TabComponent extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = { activeTab: 0 };
+    }
+
+    onTabChange = (index) => {
+        const { tabChanged } = this.props;
+        this.setState({ activeTab: index }, () => tabChanged(index));
+    };
+
+    render() {
+        return (
+            <div className="tab-container">
+                {this.props.tabs.map((value, index) => {
+                    return (
+                        <div
+                            key={index}
+                            onClick={() => this.onTabChange(index)}
+                            className={
```

Comment:
it's realy need to add empty string to classname? I think no, and to avoid this also use classnames package(it checked outside the pachage)
Why didn't use classnames? this case can be rewritten like this:
```
 <div
    key={index}
    onClick={() => this.onTabChange(index)}
    className=classnames(tab, {
        active: index === this.state.activeTab,
    });
>
    {value}
</div>
```

Reply:
No need of empty string , as i said i am not able to pass index to classnames function.